style.css

```diff
@@ -0,0 +1,109 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital@0;1&display=swap');
+
+html {
+  box-sizing: border-box;
+  font-size: 10px;
+  font-family: 'Poppins', sans-serif;
+  background: #ffff;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+.head-wrapper {
+  padding-bottom: 30px;
+  margin: 25px 25px 0 25px;
+}
+
+.nav-bar {
+  display: flex;
+}
+
+.nav-bar ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  flex: 10;
+}
+
+.menu {
+  display: block;
+  flex: 1;
+}
+
+.logo {
+  text-decoration: none;
+  color: #6070ff;
+  font-size: 18px;
+  font-weight: 700;
+  line-height: 20px;
+}
+
+.toogle-btn {
+  width: 35px;
+  height: 3px;
+  background-color: #6070ff;
+  margin: 6px 0;
+}
+
+.headline-wrapper {
+  padding-top: 100px;
+  padding-bottom: 100px;
+  border-bottom-left-radius: 110px;
+  background-image: url(file:///C:/Users/VS47~1/AppData/Local/Temp/header_section_background.png);
+  background-size: 100%;
```

Comment:
- [ ] This image is still broken because its path points to a folder on your personal computer, not the repository, Kindly edit it with a valid path as the first review suggested.

Reply:
![Vana](https://user-images.githubusercontent.com/89217512/143464816-a0c922d1-0664-4407-9200-6a142eba2571.png)

I tried fixing all the review requirements and this is the outcome. Can you please check if everything is okay, so that I can submit my last review.

Reply:
- [ ] Okey I can see you are improving!! :clap: I think the next Reviewer will approve :heavy_check_mark: